src/JestExt.ts

```diff
@@ -562,7 +459,19 @@ export class JestExt {
       }
     })
 
-    vscode.debug.startDebugging(vscode.workspace.workspaceFolders[0], configuration)
+    const workspaceFolder = vscode.workspace.workspaceFolders[0]
+    try {
+      // try to run the debug configuration from launch.json
+      await vscode.debug.startDebugging(workspaceFolder, 'vscode-jest-tests')
+    } catch {
+      // if that fails, there (probably) isn't any debug configuration (at least no correctly named one)
+      vscode.window.showWarningMessage(
+        'Debug Jest test: No correctly named debug configuration found in launch.json, starting debugging using the default configuration.'
+      )

```

Comment:
no need to show warning here as it is the default behavior when the user doesn't have any custom debug config, which is not required.

Reply:
I added it because of your "maybe we can capture error here as well and provides some helpful notes to improve the user experience…", but see the next comment.